.changeset/neat-mangos-judge.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': minor
+---
+
+Fallback to the `tap` prefetch strategy on slow connection, instead of bailing.
```

Comment:
```suggestion
Changes the fallback prefetch behavior on slow connections and when data saver mode is enabled. Instead of disabling prefetch entirely, the `tap` strategy will be used.
```
Something like this that indicates this is a change is helpful, I think!

Reply:
Looks good to me! 🙌